src/main/java/com/google/cloud/spanner/jdbc/TransactionRetryListener.java

```diff
@@ -28,14 +28,14 @@
   /** Use {@link com.google.cloud.spanner.connection.TransactionRetryListener.RetryResult} */
   @InternalApi
   @Deprecated
-  public enum RetryResult {
+  enum RetryResult {
```

Comment:
I think this would be a breaking change, no?

Reply:
No, the enum is defined inside a public interface, which makes the enum implicitly public as well. So the public keyword is superflous in this case.